img/b/beeper.com.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 68.2 69.24"><defs><linearGradient id="a" x1="31.29" x2="29.65" y1="60.89" y2="16.52" gradientTransform="matrix(1 0 0 -1 0 92)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#b547e9"/><stop offset="1" stop-color="#4958f5"/></linearGradient><linearGradient id="b" x1="38.14" x2="37.11" y1="96.74" y2="35.76" gradientTransform="matrix(1 0 0 -1 0 92)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ff9b71"/><stop offset="1" stop-color="#db48f3"/></linearGradient></defs><path fill="url(#a)" d="M48.64 31.09H7.38L.22 61.54c-1.61 6.82 6 10.26 12 5.46l6.9-5.46h22.36c8.55 0 17.08-6.82 19.06-15.23s-3.35-15.22-11.9-15.22Z"/><path fill="url(#b)" d="M26.62 0C20 0 13.46 5.25 11.94 11.73L7.39 31.09h40.93c8.73 0 17.44-7 19.46-15.55S64.36 0 55.63 0Zm-7.23 36.55 6.9-5.46H7.38c-1.6 6.82 5.95 10.26 12.01 5.46Z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25.5 32"><path fill="#2853f7" d="M0 9c0-5 4-9 9-9h7.5c5 0 9 4 9 9a9 9 0 0 1-2.8 6.5L21 17.2c2.5 1.5 4.5 3.8 4.5 6.9 0 4.6-3.8 7.9-8.4 7.9H4.5V16.8A9 9 0 0 1 0 9m9-6.5a6.5 6.5 0 1 0 0 13h3.1l-1.7 8.7 10.5-10.4A6.5 6.5 0 0 0 23 9c0-3.6-3-6.5-6.5-6.5z"/></svg>
```

Comment:
I can't find that logo version anywhere online. I only see a version where the B is white and the chat bubble is blue/purple.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25.5 32"><path fill="#fff" d="M0 9c0-5 4-9 9-9h7.5a9 9 0 0 1 6.2 15.5L21 17.2c2.5 1.5 4.5 3.8 4.5 6.9 0 4.6-3.8 7.9-8.4 7.9H4.5V16.8A9 9 0 0 1 0 9"/><path fill="#4a53f4" d="M9 2.5a6.5 6.5 0 1 0 0 13h3.1l-1.7 8.7 10.5-10.4a6.5 6.5 0 0 0-4.5-11.3"/></svg>
```